common-arcana.lic

```diff
@@ -282,10 +282,15 @@ module DRCA
     end
 
     return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'])
+    prepare_time = Time.now
 
     find_charge_invoke_stow(settings.cambrinth, settings.stored_cambrinth, settings.cambrinth_cap, settings.dedicated_camb_use, data['cambrinth'])
 
-    waitcastrt?
+    if data['prep_time']
+      pause 0.1 until checkcastrt.zero? || Time.now - prepare_time >= data['prep_time']
```

Comment:
I haven't seen `checkcastrt` used in other scripts, so ill mention that it is a built in lich function that returns the current casting roundtime as a float